corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -34,27 +34,32 @@ <h2 class="text-hq-nav-header">{% trans "Data Dictionary" %}</h2>
     <div data-bind="saveButton: saveButton"></div>
     <div class="row">
         <div class="col-md-10">
-            <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">Export to Excel</a>
-            <a class="btn btn-default" href="{% url "upload_data_dict" domain %}">Import from Excel</a>
+            <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">{% trans "Export to Excel" %}</a>
```

Comment:
There is currently an expand/collapse groups (the little +/- icons near the name of the group), should that be done differently, or is it fine as is. As far as help for this page. I was imagining we would do an internal announce, and that plus documentation on confluence would be good enough. Do you think it needs more inline explanations? It felt generally straightforward to me but I have probably lost perspective because of all the time I spend using it